File: PHP/program/Product.php
<!-- Kelas Product -->
<?php
class Product{
    private $idProduct = '';
    private $name = '';
    private $brand = '';
    private $price = 0;
    
    // Constructor
    public function __construct($idProduct='', $name='', $brand='', $price=0) {
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    // Setter untuk idProduct
    public function setIdProduct($idProduct) {
        $this->idProduct = $idProduct;
    }

    // Getter untuk idProduct
    public function getIdProduct() {
        return $this->idProduct;
    }

    // Setter untuk name
    public function setName($name) {
        $this->name = $name;
    }

    // Getter untuk name
    public function getName() {
        return $this->name;
    }

    // Setter untuk brand
    public function setBrand($brand) {
        $this->brand = $brand;
    }

    // Getter untuk brand
    public function getBrand() {
        return $this->brand;
    }

    // Setter untuk price
    public function setPrice($price) {
        $this->price = $price;
    }

    // Getter untuk price
    public function getPrice() {
        return $this->price;
    }
}
?>

File: CPP/program/Product.cpp
//import library
#include <iostream>
#include <string>

//using standar namespace
using namespace std;

class Product{ //class Product

    //Private attributes
    private:
        string idProduct; 
        string name;        
        string brand;
        int price;

        

    //public methods
    public:
        Product(){
            this->idProduct = "";
            this->name = "";
            this->brand = "";
            this->price = 0;
        }

        Product(string idProduct, string name, string brand, int price){
            this->idProduct = idProduct;
            this->name = name;
            this->brand = brand;
            this->price = price;
        }

        string get_idProduct() { return idProduct; }
        string get_name() { return name; }
        string get_brand() { return brand; }
        int get_price() { return price; }

        void set_idProduct(string idProduct) {
        this->idProduct = idProduct; 
        }
        void set_name(string name) {
        this->name = name; 
        }
        void set_brand(string brand) {
        this->brand = brand; 
        }
        void set_price(int price) {
        this->price = price; 
        }


        //destractor
        ~Product(){
            
        }
};